setup.py

```diff
@@ -23,7 +23,7 @@ def package_data(pkg, root_list):
 
 setup(
     name='xblock-image-explorer',
-    version='0.5.0',
+    version='0.7.0',
```

Comment:
Any reason why `0.7.0` and not `0.6.0`?

Reply:
Nevermind, figured it out :) Somebody forgot to update this to 0.6.0 when tagging the [v0.6.0](https://github.com/edx-solutions/xblock-image-explorer/tree/v0.6.0) release.